### DISCUSSION TITLE
Default Tab

### DISCUSSION THREAD

**Discussion by MarniRS** [runelite/runelite#15252] [Ideas]
A plugin to set your default tab. If you hop worlds your tab is always forced to the world hopper. A dropdown list to choose inventory,spellbook,prayer tab etc to have this set as your default tab after you hop worlds.
![defaulttab](https://user-images.githubusercontent.com/45274030/176980073-cf121e8c-5e4b-4667-9ea0-777f50655dfd.png)

**Comment by MarniRS** [comment]
Could be disabled in PvP scenarios incase it'd be considered as an unfair advantage there.

**Comment by damencs** [comment]
If this is deemed compliant, I can add it to the plugin hub as an external with heavy restrictions (as this was originally implemented with none).
